frame/contracts/src/wasm/runtime.rs

```diff
@@ -39,6 +39,7 @@ const TRAP_RETURN_CODE: u32 = 0x0100;
 enum SpecialTrap {
 	/// Signals that trap was generated in response to call `ext_return` host function.
 	Return(Vec<u8>),
+	OutOfGas,
```

Comment:
Would be useful to leave some words about this variant.

Reply:
Yes. Makes sense.

Reply:
Added a comment